docs/bidders/smaato.md

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
The official repository for bidder has documentation has recently changed to: https://github.com/prebid/prebid.github.io

Can you please open a separate PR for that repo instead of adding the docs to this repo? Further instructions are found here: https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#document-your-adapter

Thank you.

Reply:
ok, I was not aware. I removed this file and will make changes to the doc repo if necessary.